scripts/setup_nlp.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+
+ENV=$1
+REGION=$2
+
+if [[ $ENV != "staging" && $ENV != "prod" && $ENV != "autopush" && $ENV != "dev" && $ENV != "feeding-america" && $ENV != "tidal" && $ENV != "karnataka" ]]; then
+  echo "First argument should be 'staging' or 'prod' or 'autopush' or 'dev' or 'feeding-america' or 'tidal' or 'karnataka'"
+  exit
+fi
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+ROOT="$(dirname "$DIR")"
+
+if [[ $REGION == "" ]]; then
+  REGION=$(yq eval '.region.primary' $ROOT/deploy/gke/$ENV.yaml)
+fi
+
+PROJECT_ID=$(yq eval '.project' $ROOT/deploy/gke/$ENV.yaml)
+PROJECT_NUMBER=$(gcloud projects list --filter="${PROJECT_ID}" --format="value(PROJECT_NUMBER)")
+
+MODEL_NAME=$(yq eval '.model_name' $ROOT/deploy/gke/$ENV.yaml)
+NETWORK_NAME=$(yq eval '.network_name' $ROOT/deploy/gke/$ENV.yaml)
+
+MODEL_BUCKET="gs://${PROJECT_ID}-modelstore"
+MODEL_PATH="${MODEL_BUCKET}/${MODEL_NAME}"
+
+if ! gsutil -q stat "${MODEL_PATH}/*" ; then
+  echo "Model path does not exists ${MODEL_PATH}"
+  
+  # This might fail, meaning that the bucket already exists.
```

Comment:
If bucket exists, it can not get through the "if" check?

Reply:
This checks if a model path exists in the specified bucket. If the bucket exists but a model is not there this will cause the script to exit.

I removed the bucket creation in case this fails as it might not be useful. It should be the user responsibility I guess to make sure everything is in check.